### ISSUE TITLE
Exception thrown while executing Spotbugs 4.0.5.

### ISSUE THREAD

**Issue by linusjf** [mebigfatguy/fb-contrib#401]
```
[INFO] Fork Value is true
     [java] Exception in thread "main" edu.umd.cs.findbugs.ba.AnalysisException: Exeption was thrown during analysis
     [java]     at edu.umd.cs.findbugs.FindBugs2.lambda$analyzeApplication$3(FindBugs2.java:1125)
     [java]     at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
     [java]     at edu.umd.cs.findbugs.FindBugs2.analyzeApplication(FindBugs2.java:1118)
     [java]     at edu.umd.cs.findbugs.FindBugs2.execute(FindBugs2.java:309)
     [java]     at edu.umd.cs.findbugs.FindBugs.runMain(FindBugs.java:395)
     [java]     at edu.umd.cs.findbugs.FindBugs2.main(FindBugs2.java:1231)
     [java] Caused by: java.util.concurrent.ExecutionException: java.lang.AssertionError: frame_type field doesn't exist
     [java]     at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122)
     [java]     at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191)
     [java]     at edu.umd.cs.findbugs.FindBugs2.lambda$analyzeApplication$3(FindBugs2.java:1120)
     [java]     ... 5 more
     [java] Caused by: java.lang.AssertionError: frame_type field doesn't exist
     [java]     at edu.umd.cs.findbugs.StackMapAnalyzer.lambda$static$0(StackMapAnalyzer.java:150)
     [java]     at java.base/java.security.AccessController.doPrivileged(Native Method)
     [java]     at edu.umd.cs.findbugs.StackMapAnalyzer.<clinit>(StackMapAnalyzer.java:142)
     [java]     at edu.umd.cs.findbugs.StackMapAnalyzer$StackMapAnalysisFactory.analyze(StackMapAnalyzer.java:66)
     [java]     at edu.umd.cs.findbugs.StackMapAnalyzer$StackMapAnalysisFactory.analyze(StackMapAnalyzer.java:58)
     [java]     at edu.umd.cs.findbugs.classfile.impl.AnalysisCache.analyzeMethod(AnalysisCache.java:368)
     [java]     at edu.umd.cs.findbugs.classfile.impl.AnalysisCache.getMethodAnalysis(AnalysisCache.java:321)
     [java]     at edu.umd.cs.findbugs.OpcodeStack.getJumpInfoFromStackMap(OpcodeStack.java:3075)
     [java]     at edu.umd.cs.findbugs.OpcodeStack.resetForMethodEntry(OpcodeStack.java:3030)
     [java]     at com.mebigfatguy.fbcontrib.collect.CollectStatistics.visitCode(CollectStatistics.java:140)
     [java]     at org.apache.bcel.classfile.Code.accept(Code.java:131)
     [java]     at edu.umd.cs.findbugs.visitclass.PreorderVisitor.doVisitMethod(PreorderVisitor.java:315)
     [java]     at edu.umd.cs.findbugs.visitclass.PreorderVisitor.visitJavaClass(PreorderVisitor.java:397)
     [java]     at org.apache.bcel.classfile.JavaClass.accept(JavaClass.java:213)
     [java]     at edu.umd.cs.findbugs.BytecodeScanningDetector.visitClassContext(BytecodeScanningDetector.java:38)
     [java]     at com.mebigfatguy.fbcontrib.collect.CollectStatistics.visitClassContext(CollectStatistics.java:108)
     [java]     at edu.umd.cs.findbugs.DetectorToDetector2Adapter.visitClass(DetectorToDetector2Adapter.java:76)
     [java]     at edu.umd.cs.findbugs.FindBugs2.lambda$null$1(FindBugs2.java:1108)
     [java]     at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
     [java]     at edu.umd.cs.findbugs.CurrentThreadExecutorService.execute(CurrentThreadExecutorService.java:86)
     [java]     at java.base/java.util.concurrent.AbstractExecutorService.invokeAll(AbstractExecutorService.java:242)
     [java]     ... 4 more
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
```

Fb-contrib crashes with above exception breaking build with Spotbugs 4.0.5.

**Comment by nhojpatrick** [comment]
I was about to raise the same issue.

**Comment by nhojpatrick** [comment]
Tried with both;
```
   GroupId: com.mebigfatguy.fb-contrib
ArtifactId: fb-contrib
   Version: 7.4.7
```

and;
```
   GroupId: com.mebigfatguy.sb-contrib
ArtifactId: sb-contrib
   Version: 7.4.7
```

**Comment by linusjf** [comment]
Aren't these the same? Or should users be using only sb-contrib given that Findbugs is now SpotBugs? I've reverted back to 4.0.4 for now. At least, that doesn't crash SpotBugs. The other option is to omit the sb-contrib plugin.

**Comment by ThrawnCA** [comment]
The `fb-contrib` artefact is for FindBugs; don't use it with SpotBugs.

**Comment by linusjf** [comment]
This appears to have been fixed with Release 4.0.6. 

https://github.com/spotbugs/spotbugs/blob/4.0.6/CHANGELOG.md

Closing issue...
